terraform/main.tf

```diff
@@ -0,0 +1,550 @@
+      DATABASE_NAME     = var.database_name
+      DATABASE_PASSWORD = var.database_password
+      DATABASE_PORT     = var.database_port
+      DATABASE_USERNAME = var.database_username
+      DATABASE_IP       = aws_db_instance.media-sentiment-rds.address
+      ACCESS_KEY_ID     = var.access_key
+      SECRET_ACCESS_KEY = var.secret_key
+    }
+  }
+}
+
+# RSS Scheduler
+
+resource "aws_iam_role" "media-sentiment-rss-scheduler-role" {
+  name = "media-sentiment-rss-scheduler-role"
+  assume_role_policy = jsonencode({
+  Version = "2012-10-17",
+  Statement = [
+    {
+      Effect = "Allow",
+      Principal = {
+        Service = "scheduler.amazonaws.com"
+      },
+      Action = "sts:AssumeRole"
+    }
+  ]
+})
+  inline_policy {
+    name = "media-sentiment-rss-inline-policy"
+
+    policy = jsonencode({
+	Version= "2012-10-17",
+	Statement = [
+		{
+			Effect= "Allow",
+			Action= "lambda:InvokeFunction"
+      Resource= [
+                "arn:aws:lambda:eu-west-2:129033205317:function:media-sentiment-rss-lambda:*",
+                "arn:aws:lambda:eu-west-2:129033205317:function:media-sentiment-rss-lambda"
+            ]
+		},
+    ]
+})
+}
+}
+
+resource "aws_scheduler_schedule" "media-sentiment-rss-pipeline-scheduler" {
+  name                         = "media-sentiment-rss-pipeline-scheduler"
+  schedule_expression_timezone = "Europe/London"
+  description                  = "Schedule to run ETL pipeline every hour"
+  state                        = "ENABLED"
+  
+  flexible_time_window {
+    mode = "OFF"
+  }
+
+  schedule_expression = "cron(0 * * * ? *)"
+
+  target {
+    arn      = "arn:aws:lambda:eu-west-2:129033205317:function:media-sentiment-rss-lambda"
+    role_arn = "arn:aws:iam::129033205317:role/media-sentiment-rss-scheduler-role"
+  }
+}
+
+# Media Sentiment RDS
+
+resource "aws_security_group" "media-sentiment-rds-sg" {
+  vpc_id = "vpc-0e0f897ec7ddc230d"
+  name = "media-sentiment-rds-sg"
+  ingress {
+    from_port = 5432
+    to_port = 5432
+    protocol = "tcp"
+    cidr_blocks = ["86.155.163.236/32"]
+  }
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+resource "aws_db_instance" "media-sentiment-rds" {
+  identifier="media-sentiment-rds"
+  allocated_storage    = 20
+  db_name              = var.database_name
+  engine               = "postgres"
+  engine_version       = "15"
+  instance_class       = "db.t3.micro"
+  username             = var.database_username
+  password             = var.database_password
+  parameter_group_name = "default.postgres15"
+  skip_final_snapshot  = true
+  performance_insights_enabled = false
+  db_subnet_group_name = "public_subnet_group"
+  publicly_accessible = true
+  vpc_security_group_ids = ["${aws_security_group.media-sentiment-rds-sg.id}"]
```

Comment:
We can set this to resource.aws_security_group [resource.aws_security_groupmedia-sentiment-rds-sg.id] to keep the formatting consistent with the rest of the code